$NetBSD: patch-lib_stdio_in_h,v 1.1 2014/08/08 12:20:06 makoto Exp $

(Ubuntsu 13.04, 14.04)
In file included from localcharset.c:30:0:
./stdio.h:410:1: error: 'gets' undeclared here (not in a function)
 _GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 ^

--- lib/stdio.in.h.orig	2010-04-06 18:34:33.000000000 +0900
+++ lib/stdio.in.h	2014-08-08 21:12:21.488160526 +0900
@@ -139,7 +139,9 @@ _GL_WARN_ON_USE (fflush, "fflush is not
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
 #undef gets
+#ifndef __linux
 _GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
+#endif
 
 #if @GNULIB_FOPEN@
 # if @REPLACE_FOPEN@
